corehq/form_processor/models.py

```diff
@@ -208,6 +208,12 @@ class XFormInstanceSQL(PartitionedModel, models.Model, RedisLockableMixIn, Attac
     # for compatability with corehq.blobs.mixin.DeferredBlobMixin interface
     persistent_blobs = None
 
+    # form meta properties
+    time_end = models.DateTimeField(null=True, blank=True)
+    time_start = models.DateTimeField(null=True, blank=True)
+    commcare_version = models.CharField(max_length=8, blank=True, null=True)
+    build_version = models.PositiveIntegerField(null=True, blank=True)
```

Comment:
Wondering if this shouldn't be called `app_version`. I don't have strong feelings about it but it would make it more consistent with `app_id`

Reply:
Changed it.